fflas-ffpack/fflas/fflas_freduce.inl

```diff
@@ -80,14 +77,6 @@ namespace FFLAS { namespace vectorised { /*  for casts (?) */
         A = A < min ? A + B : A;
         A = A > max ? A - B : A;
         return A;
-
-//        return fmod(A,B);
-    }
-
-    template<size_t K, size_t MG>
```

Comment:
Why did you remove this specialization?

Reply:
I think it was only used internally and not required anymore. A quick dirty check by instrumenting this function and calling test-fgemm seems to confirm that. Given that this function just calls RecInt's mod, I don't think there's any point in re-adding it as a specialization and ensuring that it's called instead of defaulting to generic code?